bacon.toml

```diff
@@ -0,0 +1,72 @@
+# This is a configuration file for the bacon tool
+#
+# Bacon repository: https://github.com/Canop/bacon
+# Complete help on configuration: https://dystroy.org/bacon/config/
+# You can also check bacon's own bacon.toml file
+#  as an example: https://github.com/Canop/bacon/blob/main/bacon.toml
+
+default_job = "check"
+
+[jobs.check]
+command = ["cargo", "check", "--color", "always"]
+need_stdout = false
+
+[jobs.check-all]
+command = [
+    "cargo",
+    "check",
+    "--all-targets",
+    "--all-features",
```

Comment:
This is the only added line from the default (aside from autoformatting this file using vscode)